test/integration/narrow/routes/enrich-request.test.js

```diff
@@ -37,8 +37,9 @@ describe('Enrich request test', () => {
     marketingYear: 2022,
     currency: 'GBP',
     schedule: 'M12',
-    dueDate: '2021-08-15',
+    dueDate: '2021-10-25',
     value: 15000,
+    received: '2021-10-25',
```

Comment:
Can you add some tests around making sure `daysWaiting` is displayed correctly?

Reply:
added tests to `get-payment-request.test.js`